js/app.js

```diff
@@ -1,30 +1,23 @@
-// wait for DOM to load before running JS
 $(document).ready( function() {
-
-  // check to make sure JS is loaded
   console.log('JS is loaded!');
-//var url_link="https://developers.zomato.com/api/v2.1/cities";
-var url_link="https://api.tvmaze.com/shows?q=";
-  // your code here
-$("#search_btn").on('click',function(){
-//url_link=url_link+"?q="+$("#input_id").val();
-  alert(url_link);
-  //  e.preventDefault();
+  // indentation is off in this file – please reformat
+var url_link="https://api.tvmaze.com/shows";
+$("#search_btn").on('click',function(e){
+  e.preventDefault();
```

Comment:
Always pass in the event in the callback for a click handler (in this case, `e`).

If you do not run `e.preventDefault()`, you will refresh the page every time you submit the form, which means none of the code after this line will even run.